assembl/static2/js/app/components/debate/common/sentiments.jsx

```diff
@@ -7,7 +9,31 @@ import { inviteUserToLogin, displayModal } from '../../../utils/utilityManager';
 import sentimentDefinitions from './sentimentDefinitions';
 import ResponsiveOverlayTrigger from '../../common/responsiveOverlayTrigger';
 
-const Sentiment = ({ sentimentCounts, mySentiment, sentiment, client, isSelected, postId, placement, isPhaseCompleted }) => {
+type SentimentProps = {
+  client: ApolloClient,
+  isPhaseCompleted: boolean,
+  isSelected: boolean,
+  mySentiment: ?string,
+  placement: string,
+  postId: string,
+  sentiment: Object, // TODO:
```

Comment:
TODO was about the Object shape I guess

Reply:
yeah, I added lots of flow types that were out of the scope of the US so I skiped some. I will try to do this one ;-)

Reply:
done !